portal-web/test/functional/com/liferay/portalweb/macros/KBFolder.macro

```diff
@@ -183,6 +183,43 @@ definition {
 		}
 	}
 
+	macro viewFolderInfo {
+		AssertTextEquals.assertPartialText(
+			key_entryTitle = "${kbFolderName}",
+			locator1 = "KnowledgeBase#DESCRIPTIVE_VIEW_ENTRY_INFO",
+			value1 = "${entryDefaultInfo}");
+
+		AssertTextNotEquals.assertNotPartialText(
+			key_entryTitle = "${kbFolderName}",
+			locator1 = "KnowledgeBase#DESCRIPTIVE_VIEW_ENTRY_INFO",
+			value1 = "${folderQuantity} Folders");
+
+		AssertTextNotEquals.assertNotPartialText(
+			key_entryTitle = "${kbFolderName}",
+			locator1 = "KnowledgeBase#DESCRIPTIVE_VIEW_ENTRY_INFO",
+			value1 = "${articleQuantity} Articles");
```

Comment:
Please use VerifyElementPresent and VerifyElementNotPresent so the test won't get stuck in any of these steps if it fails

Reply:
Hi @austinchiang Those assertions don't fit this test. Because the path (`KnowledgeBase#DESCRIPTIVE_VIEW_ENTRY_INFO`) will get all information on the entry. We can only check which texts are present and which are not.

Reply:
@Jeremy-Ch we can't form a more specific xpath?

Reply:
@austinchiang Unless we add the precise path for every info element. I think we can keep using the current assertion. Because they are used to assert elements that have been removed. They have a low probability of affecting the test pass rate. If you think we have to make some changes to this. I can do more research and work on another ticket.